File: docs/code-examples/point3d_simple.rs
//! Log some very simple points.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec = rerun::RecordingStreamBuilder::new("rerun_example_points3d_simple").stdout()?;

    rec.log(
        "points",
        &rerun::Points3D::new([(0.0, 0.0, 0.0), (1.0, 1.0, 1.0)]),
    )?;

    Ok(())
}

File: docs/code-examples/point3d_simple.cpp
// Log some very simple points.

#include <rerun.hpp>

int main() {
    const auto rec = rerun::RecordingStream("rerun_example_points3d_simple");
    rec.spawn().exit_on_failure();

    rec.log("points", rerun::Points3D({{0.0f, 0.0f, 0.0f}, {1.0f, 1.0f, 1.0f}}));
}
